pyaedt/edb_core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -0,0 +1,1700 @@
+from pyaedt.edb_core.general import convert_py_list_to_net_list
+from pyaedt.generic.general_methods import generate_unique_name
+from pyaedt.generic.general_methods import pyaedt_function_handler
+
+
+class EdbFrequencySweep(object):
+    """Manages EDB methods for frequency sweep."""
+
+    def __init__(self, sim_setup, frequency_sweep=None, name=None, edb_sweep_data=None):
+
+        self._sim_setup = sim_setup
+
+        if edb_sweep_data:
+            self._edb_sweep_data = edb_sweep_data
+            self._name = self._edb_sweep_data.Name
+        else:
+            if not name:
+                self._name = generate_unique_name("sweep")
+            else:
+                self._name = name
+            self._edb_sweep_data = self._sim_setup._edb.simsetupdata.SweepData(self._name)
+            self.set_frequencies(frequency_sweep)
+
+    @pyaedt_function_handler()
+    def _update_sweep(self):
+        """Update sweep."""
+        self._sim_setup._edb_sim_setup_info.SweepDataList.Clear()
+        for el in list(self._sim_setup.frequency_sweeps.values()):
+            self._sim_setup._edb_sim_setup_info.SweepDataList.Add(el._edb_sweep_data)
+        self._sim_setup._edb_sim_setup_info.SweepDataList.Add(self._edb_sweep_data)
+        return self._sim_setup._update_setup()
+
+    @property
+    def name(self):
+        """Get name of this sweep."""
+        return self._edb_sweep_data.Name
+
+    @name.setter
+    def name(self, value):
+        """Set name of this sweep"""
+        self._edb_sweep_data.Name = value
+        self._update_sweep()
+
+    @property
+    def sweep_type(self):
+        """Get sweep type."""
+        return
+
+    @property
+    def frequencies(self):
+        """Get list of frequencies points."""
+        return list(self._edb_sweep_data.Frequencies)
+
+    @property
+    def adaptive_sampling(self):
+        """Use adaptive sampling.
```

Comment:
```suggestion
        """Whether adaptive sampling is used.
```
Properties are generally to be described in terms of a noun--but this has a Boolean return, so I'm not sure if my change is correct.